Formula/emqx.rb

```diff
@@ -0,0 +1,46 @@
+class Emqx < Formula
+  desc "MQTT broker for IoT"
+  homepage "https://www.emqx.io/"
+  url "https://github.com/emqx/emqx/archive/refs/tags/v5.0.9.tar.gz"
+  sha256 "bc5d193a02840b1d4cb5f6c9f047003b442c5a0b4a3ccb1cfeedafacf04ff82e"
+  license "Apache-2.0"
+  head "https://github.com/emqx/emqx.git", branch: "master"
+
+  def dashboard_version
+    "1.1.0"
+  end
+
+  def emqx_version
+    "5.0.9"
+  end
+
+  depends_on "autoconf"  => :build
+  depends_on "automake"  => :build
+  depends_on "cmake"     => :build
+  depends_on "coreutils" => :build
+  depends_on "erlang@24" => :build
+  depends_on "freetds"   => :build
+  depends_on "libtool"   => :build
+  depends_on "openssl@1.1"
+
+  uses_from_macos "curl"  => :build
+  uses_from_macos "unzip" => :build
+  uses_from_macos "zip"   => :build
+
+  def install
+    ENV["PKG_VSN"] = emqx_version
+    ENV["PROFILE"] = "emqx"
+    ENV["EMQX_DASHBOARD_VERSION"] = "v#{dashboard_version}"
```

Comment:
Aren't these versions defined in the tarbal you download?

Reply:
during build we download dashboard files and integrate it into the application